libretroshare/src/services/p3banlist.cc

```diff
@@ -1304,4 +1299,4 @@ int p3BanList::printBanSources_locked(std::ostream &out)
     return true ;
 }
 
-
+RsBanList::~RsBanList() = default;
```

Comment:
In terms of how much size is used and code clarity, I prefer the "virtual ~RsBanList() {}" in the .h: it removes the need to go into the cpp file to figure out what this destructor is actually doing to discover that it does nothing.

Reply:
...and it spares the linking job to the compiler too.

Reply:
I do prefer too (`=default` can be used in the header too), but the compiler print a warning about the vtable being emitted in every compilation unit that includes the headed if no method is defined outside the header, so moved the empty destructor in an the implementation file